src/System.Net.Http.WinHttpHandler/tests/project.lock.json

```diff
@@ -173,10 +173,10 @@
           "lib/dotnet/System.IO.FileSystem.Primitives.dll": {}
         }
       },
-      "System.Linq/4.0.0-beta-22816": {
+      "System.Linq/4.0.0-beta-22703": {
         "dependencies": {
-          "System.Collections": "4.0.10-beta-22816",
-          "System.Runtime": "4.0.20-beta-22816"
+          "System.Collections": "4.0.0-beta-22703",
+          "System.Runtime": "4.0.0-beta-22703"
```

Comment:
Looks like the bug is in the project.json.  https://github.com/dotnet/corefx/blob/master/src/System.Net.Http.WinHttpHandler/tests/project.json
This is using versions like 4.0.0-* instead of 4.0.0-beta-*.  It's odd that nuget is behaving this way but we should update the project to be consistent with the others.